electricpy/constants.py

```diff
@@ -28,6 +28,8 @@
 NAN = float('nan')
 VLLcVLN = _c.rect(_np.sqrt(3),_np.radians(30)) # Conversion Operator
 ILcIP = _c.rect(_np.sqrt(3),_np.radians(30)) # Conversion Operator
+watts_per_hp = 745.699872
```

Comment:
Probably best if we start (I say start, because I'm fairly certain that I haven't done it consistently elsewhere) making these constants all UPPERCASE to be more consistent with Python name standards.

- [ ] change: `watts_per_hp` -> `WATTS_PER_HP`
- [ ] change: `kwh_per_btu` -> `KWH_PER_BTU`